src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -68,6 +68,7 @@ Overall, proof-of-stake, as it is implemented on Ethereum, has been demonstrated
 - [Why Proof of Stake (Nov 2020)](https://vitalik.ca/general/2020/11/06/pos2020.html) _Vitalik Buterin_
 - [Proof of Stake: How I Learned to Love Weak Subjectivity](https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/) _Vitalik Buterin_
 - [A Proof of Stake Design Philosophy](https://medium.com/@VitalikButerin/a-proof-of-stake-design-philosophy-506585978d51) _Vitalik Buterin_
+- [Proof of Stake — What is and how does it work?](https://medium.com/nerd-for-tech/proof-of-stake-what-is-and-how-does-it-work-1c8bd7f00303) _Henrique Centieiro_
```

Comment:
I don't think this adds anything that isn't already explained here or better in other articles. Also not keen on promoting anything talking about "Ethereum 2.0" or staking with a centralized provider (hello affiliate links) without providing the cons or at least an alternative. I wouldn't be in favour of adding this.